catkit/emulators/iris_ao_controller.py

```diff
@@ -48,7 +48,13 @@ def convert_command_to_poppy_surface(dm_command_dict):
 
         # Setting the simulated IrisAO means setting each actuator individually
         for seg, values in convert_command_to_poppy_surface(new_surface).items():
-            self.set_actuator(seg-1, values[0] * u.um, values[1] * u.mrad, values[2] * u.mrad)  # TODO: double-check the -1 here, meant to correct for different segment names
+            # The IrisAO hardware component and the poppy hex DM object have flipped
+            # x/y coordinate axes, so we need to feed what is called "tilt" on the IrisAO
+            # into the "tip" argument from poppy and vice versa.
+            piston = values[0] * u.um
+            tip = values[2] * u.mrad
+            tilt = values[1] * u.mrad
+            self.set_actuator(seg-1, piston, tip, tilt)  # TODO: double-check the -1 here, meant to correct for different segment names
```

Comment:
Is the "# TODO: double check" comment needed any longer? Since you've tested this works and we're commanding the correct segments, seems to me probably not.

Reply:
Yes it works, but I had literally just thrown the -1 in there when I got an error initially, and didn't think about it further as to why I needed to add it to make it work. I can remove it if you want? But I will keep a personal note to come back here to see what exactly is happening. Let me know.

Reply:
if you still don't understand it then ok leave the comment in for now.   But surely this is just a different convention for 0-based one 1-based indices. Poppy numbers segments starting from 0. I infer the Iris code does not, and starts from 1 instead. Right?

Reply:
Yes, it will be something like this. The reason I want to double-check is that to my understanding, the commands are already taking care of that. So I just want to make sure it is really this.